client/cli/src/lib.rs

```diff
@@ -459,7 +460,12 @@ enum Command {
     },
 
     /// Upload the (encrypted) snapshot of mint notes to federation
-    Backup,
+    Backup {
+        #[clap(long = "metadata")]
+        /// Backup metadata, encoded as `key=value` (use `--metadata=key=value`,
+        /// possibly multiple times)
+        metadata: Vec<String>,
```

Comment:
nit: can this be directly parsed into the semantically more correct `*Map<String, String>` structure?

Reply:
I looked a little bit can clap do it, and I didn't see a good way. I'm adding a `TODO` to clean up later. Not a big deal, I think.